### PR TITLE
minor fixes to tofu role

### PR DESCRIPTION
**Pull request by eshulman2**
made some minor fixes to make tofu role easier to use

As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date:
  - [x] README in the role
  - [x] Content of the docs/source is reflecting the changes

### CODE REVIEW

File: roles/tofu/tasks/plan.yml [hunk 1]
```diff
@@ -14,13 +14,13 @@
...
 
-- name: "Tofu Action: '{{ cifmw_tofu_plan_state }}'"
+- name: "Tofu Action"
   environment:
     TF_CLI_ARGS: '-no-color'
     TF_IN_AUTOMATION: 'true'
   vars:
     tf_project_path: "{{ cifmw_tofu_remote_execution | ternary(cifmw_tofu_project_temp_dir['path'], cifmw_tofu_project_path) }}"
-  community.general.cloud.misc.terraform:
+  terraform:
```
Comment (pablintino) [inline roles/tofu/tasks/plan.yml:23]: Why is this not working? The Ansible recommendation is to use the FQN of each/role module.

Comment (cjeanner) [reply roles/tofu/tasks/plan.yml:23]: @eshulman2 hello, care to check that comment out? FQCN are mandatory in ansible, especially for non-builtin modules.

**Review by cjeanner — COMMENTED** [review]
@eshulman2 hello, is this still relevant/needed? If not, care to close it? If yes, care to address the comment about FQCN? Thanks!

File: roles/tofu/tasks/plan.yml (hunk 1 above)
Comment (eshulman2) [reply roles/tofu/tasks/plan.yml]: changed

**Comment by eshulman2** [comment]
> @eshulman2 hello, is this still relevant/needed? If not, care to close it? If yes, care to address the comment about FQCN? Thanks!

@cjeanner sorry for the delay with fixing I wanted to verify on a working deployment and only now got one. should be ok now

**Pull request merged**